src/token/handler.ts

```diff
@@ -246,7 +246,7 @@ export class Login7TokenHandler extends TokenHandler {
   declare connection: Connection;
 
   declare fedAuthInfoToken: FedAuthInfoToken | undefined;
-  declare routingData: { server: string, port: number } | undefined;
+  declare routingData: { server: string, port: number, login7server: string } | undefined;
```

Comment:
I'd rather store the server and just the instancename here, and then just join them together when the combination of them is required?

Reply:
@arthurschreiber Agreed - switched to using instance